scala/private/common.bzl

```diff
@@ -58,11 +63,16 @@ def _collect_jars_when_dependency_analyzer_is_off(
         else:
             print("ignored dependency, has no JavaInfo: " + str(dep_target))
 
+        if (not plus_one_deps_is_off) and (PlusOneDeps in dep_target):
+            plus_one_deps_compile_jars.append(
+                depset(transitive = [dep[JavaInfo].compile_jars for dep in dep_target[PlusOneDeps].direct_deps if JavaInfo in dep ])
+            )
+
     return struct(
         compile_jars = depset(transitive = compile_jars),
         transitive_runtime_jars = depset(transitive = runtime_jars),
         jars2labels = JarsToLabelsInfo(jars_to_labels = jars2labels),
-        transitive_compile_jars = depset(),
```

Comment:
huh, this is interesting. This is a change from before. We weren't really tracking transitive compile jars?

Reply:
if strict deps was off we didn't track it